pkgs/development/libraries/science/chemistry/avogadrolibs/default.nix

```diff
@@ -0,0 +1,67 @@
+{ lib, stdenv, fetchFromGitHub, cmake, zlib, eigen, libGL, doxygen, spglib
+, mmtf-cpp, glew, python3, libarchive, libmsym, msgpack, qt5, wrapQtAppsHook
+}:
+
+let
+  pythonWP = python3.withPackages (p: with p; [ openbabel-bindings numpy ]);
```

Comment:
this doesn't seem to be used anywhere

Reply:
You are right. Was introduced when I've searched for the OpenBabel bindings. Has been removed.